spec/models/comment_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
```

Comment:
- [ ] Currently, I am getting these errors when trying to run your tests. So kindly fix that as well so that all your tests pass as expected 😉 
- Error Message
![image](https://github.com/ctokoli/Blog/assets/102699958/7c8942f2-16db-4391-946b-09c51ecf709b)